tests/test_ilastik/test_applets/dataSelection/testOpDataSelection.py

```diff
@@ -877,6 +883,12 @@ def test_load_precomputed_chunks_over_http(self, graph, monkeypatch):
         loaded_scale1 = op.Image[:].wait()
         assert numpy.allclose(loaded_scale1, self.IMAGE_ORIGINAL)
 
+    def test_save_scale_change(self, op_and_dataset):
```

Comment:
Technically this doesn't test saving/loading. It's more like `test_scale_change_updates_dataset_info`... but maybe too much of a handful.